app/models/ark.rb

```diff
@@ -21,6 +21,8 @@ def self.find(ezid)
   # @param [ezid] [String] the EZID being validated
   # @return [Boolean]
   def self.valid?(ezid)
+    # Always consider test ARKs valid
+    return true if ezid.start_with?(EZID_TEST_SHOULDER)
```

Comment:
@jrgriffiniii - I still had to add this to the check since we have ARKs with the "ark:/99999" prefix. I also renamed the constant from `EZID_INVALID_SHOULDER` to `EZID_TEST_SHOULDER` otherwise the code will read weird if it says "return true when it has invalid shoulder"

Reply:
I am sorry for not addressing this case.